distributions/Lakka/options

```diff
@@ -383,6 +383,7 @@
 "/tmp/cores/genesis_plus_gx_libretro.so;"\
 "/tmp/cores/genesis_plus_gx_libretro.so;"\
 "/tmp/cores/genesis_plus_gx_libretro.so;"\
+"/tmp/cores/gearsystem_libretro.so;"\
```

Comment:
this line has to be removed, otherwise the array/list is not correct. in this list (`RA_PLAYLIST_CORES`) only one core per system/playlist (`RA_PLAYLIST_NAMES`) is allowed. when you added the gearsystem core, you have shifted all cores below this line to different playlist/system. you have modified the default core for "Sega - PICO", if that was your intention, then remove the line below, but I would suggest keeping the default cores as they are - so just remove this line.